docs/using-ngrok-with/k8s.md

```diff
@@ -65,7 +65,7 @@ To started with the ngrok Ingress Controller for Kubernetes:
       --set credentials.authtoken=$NGROK_AUTHTOKEN
     ```
 
-1. Create a manifest file (for example `ngrok-manifest.yaml`) with the following contents. You will need to replace the `NGROK_DOMAIN` on line 45 with your own custom value. This is the URL you will use to access your service from anywhere. If you're on a free account, it must be a subdomain of either `ngrok-free.app` or `ngrok-free.dev` (for example, `username-loves-ingress.ngrok-free.app`). For paid accounts, you can use a custom domain or a subdomain of `ngrok.app` or `ngrok.dev` (for example, `username-loves-ingress.ngrok.app` or `k8s.example.com`).
+1. Create a manifest file (for example `ngrok-manifest.yaml`) with the following contents. You will need to replace the `NGROK_DOMAIN` on line 45 with your own custom value. This is the URL you will use to access your service from anywhere. If you're on a free account, it must be on a static subdomain which you can claim by logging into your account and following the instructions on the claim static subdomain banner. For paid accounts, you can use a custom domain or a subdomain of `ngrok.app` or `ngrok.dev` (for example, `username-loves-ingress.ngrok.app` or `k8s.example.com`).
```

Comment:
Question: Is the banner going to be there long term, or should we direct them to docs or offer more detail?

Reply:
Once we push out the blog post will switch over to pointing at that